accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a project workspaces.
+
+# Motivation
+
+With the addition of the Workspace feature, it will become handy to be able to execute commands inside other workspaces than the one in the current directory.
+
+# Detailed design
+
+This RFC suggests to add the two following commands:
+
+```
+$> yarn workspace <project-name> pwd
+$> yarn workspace <project-name> <command-name> ...
```

Comment:
One of the things I'd get a bit worries about here is overloading the same command space. There's three different things that fit into this space broken down into two groups:

- `yarn workspace <name> <command/run-script>` which matches `yarn <command/run-script>`
- `yarn workspace <name> <system-command>` (i.e. `yarn workspace <name> pwd`)

That's why I would like to see a new global command `exec` (which is similar to `lerna exec`)

| project-level | workspace-level |
| --- | --- |
| `yarn <command/run-script>` | `yarn workspace <name> <command/run-script>` |
| `yarn run <run-script>` | `yarn workspace run <run-script>` |
| `yarn exec <system-command>` | `yarn workspace exec <system-command>` |